pyramid_swagger/exceptions.py

```diff
@@ -2,10 +2,10 @@
 import sys
 
 import six
-from pyramid.httpexceptions import HTTPClientError, HTTPInternalServerError
+from pyramid.httpexceptions import HTTPBadRequest, HTTPInternalServerError
 
 
-class RequestValidationError(HTTPClientError):
+class RequestValidationError(HTTPBadRequest):
```

Comment:
Yeah, I think that should be sufficient.  If people were catching `HTTPClientError` they would have basically had a catch all exception handler because its the base class for pyramid http client exception.  But even with that, `HTTPBadRequest` subclasses it so their code should still work and shouldn't be considered an interface change